TangoRosStreamer/app/src/main/java/eu/intermodalics/tango_ros_streamer/activities/RunningActivity.java

```diff
@@ -384,11 +385,11 @@ private void saveUuidsNamestoHashMap() {
         mTangoServiceClientNode.callGetMapUuidsService();
     }
 
-    private void getTangoPermission(String permissionType) {
+    private void getTangoPermission(String permissionType, int requestCode) {
         Intent intent = new Intent();
         intent.setAction(REQUEST_TANGO_PERMISSION_ACTION);
         intent.putExtra(EXTRA_KEY_PERMISSIONTYPE, permissionType);
-        startActivityForResult(intent, REQUEST_CODE_TANGO_PERMISSION);
+        startActivityForResult(intent, requestCode);
```

Comment:
Following current Tango documentation ([here](https://developers.google.com/tango/apis/java/java-user-permissions)), this can be done in a simpler way:
```
startActivityForResult(
    Tango.getRequestPermissionIntent(Tango.PERMISSIONTYPE_ADF_LOAD_SAVE),
    Tango.TANGO_INTENT_ACTIVITYCODE);
```